<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rowsField">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="colsField">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comfortaa</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter number of rows and columns: </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>741</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="sortedMatrixLAbel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>701</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="matrixLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>371</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="sortTimeLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>175</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
